backend/app/api/endpoints/adapter/task_restore.py

```
@@ -0,0 +1,80 @@
+
+    success: bool
+    task_id: int
+    task_type: str
+    executor_rebuilt: bool
+    message: str
+
+
+@router.post("/{task_id}/restore", response_model=RestoreTaskResponse)
+def restore_task(
+    task_id: int,
+    request: RestoreTaskRequest = RestoreTaskRequest(),
```

Ruff (0.14.14)

[warning] Do not perform function call `RestoreTaskRequest` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)